src/forms/renderers/OAuth.tsx

```diff
@@ -219,7 +220,11 @@ const OAuthproviderRenderer = ({
             );
         } else if (!isEmpty(fetchAuthURL.data)) {
             // This kicks off the call and the success is handled with the onSuccess/onError
-            getAuth(fetchAuthURL.data.url, fetchAuthURL.data.state);
+            getAuth(
+                fetchAuthURL.data.url,
+                fetchAuthURL.data.state,
+                fetchAuthURL.data.code_verifier
+            );
```

Comment:
Now when we have the URL we pass along the verifier. We could probably change this to just pass the single `fetchAuthURL.data` object given how much is used but did not want to change all that for this.